modules/installation-initializing-manual.adoc

```diff
@@ -118,12 +121,12 @@ mirror the repository.
 endif::restricted[]
 +
 
-ifndef::aws-china,aws-gov,ash[]
+ifndef::aws-china,aws-gov,aws-secret,ash[]
```

Comment:
This is outside the scope of this PR, but this conditional should include `azure-gov`, too.

Reply:
@codyhoag When you have a chance, can you please take a look at this. We might be missing a condition in the Azure gov content. Can you verify? If we turns out we need it, happy to make the update while I am here.

Reply:
@mjpytlak agreed. Looks like `azure-gov` was missed there :+1: